src/dfx/src/lib/builders/mod.rs

```diff
@@ -376,6 +376,23 @@ impl BuildConfig {
     }
 }
 
+fn shrink_wasm(wasm_path: impl AsRef<Path>) -> DfxResult {
+    let wasm_path = wasm_path.as_ref();
+    let wasm = std::fs::read(wasm_path).expect("Could not read the WASM module.");
```

Comment:
Is there a reason not to use context here?
```suggestion
    let wasm = std::fs::read(wasm_path).context("Could not read the WASM module.")?;
```